src/logic/wallets/e2e-wallet/module.ts

```diff
@@ -3,8 +3,7 @@ import { WalletModule } from 'bnc-onboard/dist/src/interfaces'
+const CYPRESS_LOGO = 'https://iconape.com/wp-content/files/gj/370774/svg/370774.svg'
```

Comment:
I meant as a data-URI obviously. We shouldn't hot-link to that website.

Reply:
ah ok hehe, changed